internal/asset/helpers.go

```diff
@@ -45,10 +47,13 @@ func assetResponseMapping(a *apiv1.Asset) assetModel {
 			Memory:             types.Int64Value(int64(m.Memory)),
 			Storage:            types.Int64Value(int64(m.Storage)),
 			CpuDescription:     types.StringValue(m.CpuDescription),
-			StorageDescription: types.StringValue(m.StorageDesription), //TODO: fix typo when fixed in api
+			StorageDescription: types.StringValue(m.StorageDesription), //TODO: fix typo when moving to latest api version
```

Comment:
```suggestion
			StorageDescription: types.StringValue(m.StorageDesription),
```

This will produce a compile error anyway. No need to add a comment